docs/src/integrations/exchange.md

```diff
@@ -0,0 +1,183 @@
+# Add Solana to Your Exchange
+
+This guide describes how to add Solana's native token SOL to your cryptocurrency
+exchange.
+
+## Node Setup
+
+We highly recommend setting up at least one of your own Solana api nodes to
+give you a trusted entrypoint to the network and allow you full control over how
+much data is retained.
+
+To run an api node:
+1. [Install the Solana command-line tool suite](../cli/install-solana-cli-tools.md)
+2. Boot the node with at least the following parameters:
+```bash
+solana-validator \
+  --ledger <LEDGER_PATH> \
+  --entrypoint <CLUSTER_ENTRYPOINT> \
+  --expected-genesis-hash <EXPECTED_GENESIS_HASH> \
+  --expected-shred-version <EXPECTED_SHRED_VERSION> \
+  --rpc-port 8899 \
+  --no-voting \
+  --enable-rpc-transaction-history
+```
+
```

Comment:
Please add a sentence or two about configuring the validator to automatically restart should it exit.  systemd is one possible solution for them

Reply:
Done, see Automatic Restarts below. Let me know if you think we should be more specific about systemd, or link to the cluster repo, etc.